03_Modules/Transactions/src/module/module.ts

```diff
@@ -412,6 +305,127 @@ export class TransactionsModule extends AbstractModule {
     }
   }
 
+  private async buildTransactionEventResponse(
+    authorizations: Authorization[],
+    message: IMessage<TransactionEventRequest>,
+    stationId: string,
+    transactionId: string,
+    transactionEvent: TransactionEventRequest,
+  ): Promise<TransactionEventResponse> {
+    const transactionEventResponse: TransactionEventResponse = {
+      idTokenInfo: {
+        status: AuthorizationStatusEnumType.Unknown,
+        // TODO determine how/if to set personalMessage
+      },
+    };
+
+    if (authorizations.length !== 1) {
+      return transactionEventResponse;
+    } else if (authorizations.length > 1) {
```

Comment:
The execution never comes into this condition since it returns when `authorizations.length !== 1`. If we follow the idea in the original code, i.e., only continue when the length is 1, I think we can remove this part.

Reply:
Done!